config/jobs/kubernetes/sig-node/sig-node-presubmit-cpumanager.yaml

```diff
@@ -0,0 +1,120 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-node-e2e-cpumanager
+    optional: true
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-containerd
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+    always_run: false
+    max_concurrency: 12
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191212-28f7cbc-master
+        args:
+        - --root=/go/src
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
+        - --scenario=kubernetes_e2e
+        - -- # end bootstrap args, scenario args below
+        - --deployment=node
+        - --gcp-project=k8s-jkns-pr-node-e2e
+        - --gcp-zone=us-west1-b
+        - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
+        - --node-tests=true
+        - --provider=gce
+        - --test_args=--nodes=8 --focus="\[CPUManager\]" --skip="" --flakeAttempts=2
+        - --timeout=65m
+        - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml
```

Comment:
I have run these tests on my local setup only. 
Do you know if/where the list of available images are documented?

I can certainly change the image, and will do so. Would be good to make sure the correct image is being utilized.

Thanks for the pointer to the CI job for CPU Manager. I wasn't sure this existed!

Reply:
> Do you know if/where the list of available images are documented?

I think it is best to just reuse the images (and the config) from the `cpu-manager` test, `--node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-serial-cpu-manager.yaml`, so we avoid duplication.

Reply:
@odinuge Yes, agree. I've spent some time becoming more familiar with the CI job for cpuManager and the config for images and machine type. 

I've also got a setup to run this on gce now, so I can do more testing there instead of relying on PR jobs.....